pkg/controller/mutators/core/controller_test.go

```diff
@@ -103,10 +106,17 @@ func TestReconcile(t *testing.T) {
 	podstatus.DisablePodOwnership()
 	pod := &corev1.Pod{}
 	pod.Name = "no-pod"
-	rec := newReconciler(mgr, mSys, tracker, func() (*corev1.Pod, error) { return pod, nil })
 
-	recFn, _ := SetupTestReconcile(rec)
-	g.Expect(add(mgr, recFn)).NotTo(gomega.HaveOccurred())
+	kind := "Assign"
+	newObj := func() client.Object { return &mutationsv1alpha1.Assign{} }
+	newMutator := func(obj client.Object) (types.Mutator, error) {
+		assign := obj.(*mutationsv1alpha1.Assign)
```

Comment:
lint error here

Reply:
Disabled linting. I think we want to panic, which will expose type mismatches during testing.